packages/css/native/shared/Css_AtomicTypes.ml

```diff
@@ -2278,7 +2278,7 @@ module FontFamilyName = struct
     match x with
     | `custom value ->
       (match Std.String.get value 0 with
-      | {js|"|js} | {js|'|js} -> value
```

Comment:
Apparently this kind of usage of `js` strings never worked, see [example](https://melange.re/v2.2.0/playground/?language=OCaml&code=let+x+%3D+match+%7Bjs%7C%F0%9F%90%AB+name%7Cjs%7D+with+%7Bjs%7C%F0%9F%90%AB+name%7Cjs%7D+-%3E+%28%29+%7C+_+-%3E+assert+false&live=off).